roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -15,20 +15,29 @@ class DockerImageAvailability(OpenShiftCheck):
 
     skopeo_image = "openshift/openshift-ansible"
 
-    # FIXME(juanvallejo): we should consider other possible values of
-    # `deployment_type` (the key here). See
-    # https://github.com/openshift/openshift-ansible/blob/8e26f8c/roles/openshift_repos/vars/main.yml#L7
-    docker_image_base = {
+    deployment_image_info = {
```

Comment:
Expanding on a previous comment about making this a flat list, I was thinking something along the lines of https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_health_checker/openshift_checks/package_availability.py#L31-L70

I haven't tried to see the implications, but had a feeling it could be simpler.